lib/CPathfinder.h

```diff
@@ -21,6 +23,19 @@ struct TerrainTile;
 
 struct DLL_LINKAGE CGPathNode
 {
+	typedef EPathfindingLayer ELayer;
+
+	enum ENodeAction
```

Comment:
While looking on memory usage of this struct I noticed some issues:
1) Enumerations are ui32. With c++11 it is possible to specify underlying type, e.g:
enum ENodeAction // 4 bytes
enum ENodeAction : ui8 // 1 byte

2) Padding - Could be already fixed by Warmonger in another branch but current layout of this structure is not optimal. Better to reorder members like this:
a) pointers - size 4 or 8 bytes (64 bit system)
b) all 4-bytes fields and int3 which contains 4-byte fields in it
c) bool's and u8's, including enum's inherited from u8